core/shark_turbine/dynamo/tensor.py

```diff
@@ -53,11 +53,7 @@
 
 from ..importers.fx_importer import FxImporter
 
-DEFAULT_COMPILER_FLAGS = (
-    # Enable asynchronous calling convention.
-    "--iree-execution-model=async-external",
```

Comment:
Do we not need iree-execution-model=async-external anymore?

Reply:
No. Async functions are always generated now (with a sync wrapper for default users).